common/app/views/fragments/stylesheets.scala.html

```diff
@@ -58,8 +58,8 @@
   - Exclude IE Mobile [&(!IEMobile)]
 *@
 <!--[if (IE 9)&(!IEMobile)]>
-    @fragments.stylesheetLink(Static.css.headIE9(projectName))
-    @fragments.stylesheetLink("stylesheets/ie9.content.css")
+    @fragments.stylesheetLink(Static.css.headIE9(projectName), isCrossword)
```

Comment:
I don't know for certain that it doesn't affect those browsers, I think it's better to apply the rule consistently for crossword pages.